Flashcards.DouglasFir/Flashcards/DAO/FlashCardDao.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Flashcards.Database;
+using Flashcards.Services;
+using Flashcards.DTO;
+using Flashcards.Models;
+using Dapper;
+
+namespace Flashcards.DAO;
+
+public class FlashCardDao
+{
+    private readonly DatabaseContext _dbContext;
+
+    public FlashCardDao(DatabaseContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public void InsertNewFlashCard(FlashCardDto flashCardDto, StackDto stack)
+    {
+        try
+        {
+            FlashCard flashCard = new FlashCard
+            {
+                StackID = stack.StackID,
+                Front = flashCardDto.Front,
+                Back = flashCardDto.Back
+            };
+
+            using (var dbConnection = _dbContext.GetConnectionToFlashCards())
+            {
+                string sql = $"INSERT INTO {ConfigSettings.tbFlashCardsName} (StackID, Front, Back) VALUES (@StackID, @Front, @Back)";
```

Comment:
I'm not really sure about how you set the table name. It's not user input, so low chance of sql injection, but I still think it'll be better to also inject the table name as a parameter.

Reply:
I was changing the view/table names and was receiving an error so I went with the config file option. I reverted the flashcards file to use a hard coded table.